### DISCUSSION TITLE
How to enable AI search page in front end.

### DISCUSSION THREAD

**Discussion by victorokinda** [infiniflow/ragflow#2707] [Q&A] [answered]
I cloned the repo and used:   `docker compose -f docker-compose-gpu.yml up `; It works well but it seems to be built on an older version of the frontend. Despite pulling the latest images (both latest and dev) some of the features listed are missing e.g. AI search page, Excel to HTML etc. I have second deployment  in a different machine that has these features even though I followed the same procedure. What could i be missing ?  Latest version:
![image](https://github.com/user-attachments/assets/57afedeb-233f-4259-9017-f1f174a50d5c)
My Version: 
![image](https://github.com/user-attachments/assets/af204579-127b-42d9-9f1f-03c5ffa069bf)

**Comment by KevinHuSh** [comment, marked as answer]
You need to rebuild the frontend using:
npm build .

**Reply by victorokinda** [reply]
thanks for response. I ended up doing a fresh install, removing remnants of an old "build-from-source" and everything worked.
